src/components/controls/TextInput/TextInput.scss

```diff
@@ -91,10 +91,28 @@ $block: '.#{variables.$ns}text-input';
         flex-shrink: 0;
     }
 
+    &__additional {
```

Comment:
We already have `additional-content`. It is a little bit confusing. Maybe smth like `outer-additional-content` ?